### PR TITLE
tightened conditions for successful data build

### PR DESCRIPTION
**Pull request by tormozz48**
Now if any of doc *.md files doesn't load by any reasons the main process will be stopped with error
The same way was already been implemented for people and library loading

@tadatuta @tavriaforever @dab

### CODE REVIEW

File: src/data/index.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ module.exports = {
                 logger.info(''.toUpperCase.apply(MSG.INFO.END))
             })
             .fail(function(err) {
-                logger.error(''.toUpperCase.apply(MSG.ERROR), err)
+                logger.error(''.toUpperCase.apply(MSG.ERROR))
```
Comment (tadatuta) [inline src/data/index.js]: isn't it the same as `MSG.ERROR.toUpperCase()`?

File: src/data/snapshot/load_sources.js [hunk 2]
```diff
@@ -23,18 +23,19 @@ module.exports = function(obj) {
...
+                return vow.all(languages.map(function (lang) {
                     return analyzeMetaInformation(node, lang, collected)
                         .then(function(res) {
                             return loadMDFile(res.node, lang, res.repo);
                         })
-                        .then(function (res) {
-                            node.source[lang].url = node.source[lang].content;
-                            node.source[lang].content = res;
+                        .then(function(res) {
+                            if(res) {
```
Comment (tadatuta) [inline src/data/snapshot/load_sources.js:35]: we need some logging in case of empty `res`, don't we?

**Comment by tadatuta** [comment]
LGTM

File: src/data/snapshot/load_sources.js (hunk 2 above)
Comment (tormozz48) [inline src/data/snapshot/load_sources.js:35]: We already have provided detail logging for all cases inside loadMDFile method

**Pull request merged**